apps/docs/src/components/code/Playground.tsx

```diff
@@ -105,8 +113,16 @@ export const Playground = ({
       {/* Component preview and controls */}
       <div className="grid grid-cols-[2fr_1fr] border rounded-t-round">
         {/* Preview Area */}
+
         <div className={"grid place-items-center p-sm"}>
-          {decorator ? decorator(componentElement) : componentElement}
+          <HvProvider
```

Comment:
we need before the `div.grid` above, otherwise the UI gets messed up. we're using the styles above for the layout